package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/latticexyz/mud.git"
   },
   "scripts": {
-    "all-build": "for dir in packages/store packages/world packages/world-modules packages/cli test/mock-game-contracts e2e/packages/contracts examples/*/packages/contracts examples/multiple-namespaces templates/*/packages/contracts; do (cd \"$dir\" && pwd && pnpm build); done",
+    "all-build": "for dir in packages/store packages/world packages/explorer packages/world-modules packages/cli test/mock-game-contracts e2e/packages/contracts examples/*/packages/contracts examples/multiple-namespaces templates/*/packages/contracts; do (cd \"$dir\" && pwd && pnpm build); done",
```

Comment:
this task was originally meant to be able to generate all the codegen that gets checked into git (rather than build all packages, which we have `pnpm build` for)

does explorer have any usages of `mud build` where we'd wanna check in those files? if not, can remove this

Reply:
Ah I see, regular `build` should be good, removed to original